### PR TITLE
Add Attributes for actors

### LINKED ISSUE #11: Attributes
Create **primary** atributes for actors. See the [GDD for reference](https://uspgamedev.org/wiki/Backdoor). Basically, these are Athletics, Arcana and Engineering.

### PR DESCRIPTION
**Pull request by dread-uo**
Closes #11 .

### CODE REVIEW

**Comment by Kazuo256** [comment]
Hum, OK, maybe the issue was not properly written. Let me rephrase it.

**Comment by dread-uo** [comment]
Oh, it makes more sense. Sorry, my bad for not reading the GDD. I'm closing this.

**Comment by Kazuo256** [comment]
Or maybe it is the GDD's fault for not being accessible (aka being boring) =D.
